backend/api/v1/auth_service.go

```diff
@@ -587,7 +587,8 @@ func (s *AuthService) Login(ctx context.Context, request *v1pb.LoginRequest) (*v
 		}, nil
 	}
 
-	var accessToken, refreshToken string
+	var accessToken string
```

Comment:
no need to change this file.

Reply:
I wouldn't bother the revert this change as it makes not difference.

Reply:
It does affect the style. Since it makes no difference, please revert the change to minimize the PR.